include/glow/Graph/Nodes.h

```diff
@@ -123,6 +123,9 @@ class Placeholder : public Storage {
   /// differentiation.
   bool isTraining() const { return isTrainable_; }
 
+  /// \returns True if this placeholder is an output of \p F.
+  bool isOutput(Function *F) const;
```

Comment:
we'd similarly need is input.
Probably !isOutput should be good, unless we could make isInput more efficient in implementation

Reply:
Some nodes can be inputs and outputs. For example, quantization nodes. How do we handle them?

Reply:
Good point! Profiling nodes are kinda special (similar thing would happen with training placeholders, but we have a way to check isTraining).

one simple solution would be to have an enum for type of placeholder { in, out, inout } and return it here.
another idea is to force placeholders for profiling nodes to be trainable (since we change values and then re-read them, similar process to training).

First approach seems to be cleaner.

Reply:
Another option could be to use `isOverwrittenNthInput()` -- we use it for checking for these other cases (profiling and training), right? So we could also add a check here for if a user is using the Placeholder as an overwritten input. Then we would also need a separate `isInput()`.